demo/Titanic/__init__.robot

```diff
@@ -0,0 +1,4 @@
+*** Settings ***
```

Comment:
This file shouldn't be needed. `Titanic` is the project folder, whereas the `Titanic_scenarios` folder is the one that represents the test suite. This file is already there.

Reply:
Will remove (think it was created by accident)